src/sources/shopify/tracker/transform.js

```diff
@@ -142,7 +143,7 @@ const createMessageForEvent = (event, shopifyTopic) => {
 
 const handleCartsUpdate = async (event, metricMetadata) => {
   if (!useRedisDatabase) {
```

Comment:
can we clean up `useRedisDatabase` variable if we are touching this code wherever possible ?

Reply:
Done